navigation.js

```diff
@@ -40,8 +40,9 @@ export default function Navigation() {
   };
 
   useEffect(() => {
+    const isInStartPage = currentPage === NAV_AUTHENTICATION || currentPage == null || currentPage === undefined
```

Comment:
This line checks if the current page is in the auth (login) screen, or undefined || null (initial boot setup). This is used later to set the "navigate on auth fail" option (we don't want to re-navigate to the auth page when the user is already there)